plone-5.2.x.cfg

```diff
@@ -16,21 +16,42 @@ package-name =
 newest = false
 show-picked-versions = true
 
+plone-user = admin:admin
+
 [versions]
 # FIXME: This can be removed if Plone folks add the pin to Plone official cfgs.
 # https://github.com/collective/buildout.plonetest/issues/25
 zc.recipe.cmmi = 1.3.6
 
 [instance]
 recipe = plone.recipe.zope2instance
-user = admin:admin
-http-address = 8080
+user = ${buildout:plone-user}
+wsgi = on
```

Comment:
Is WSGI meant to be only on in python 3.
I suspect this would give slightly different results, or a different command should be run. Well, bin/test would still be the same.

Reply:
well ... I copied this from buildout.coredev where it is exactly the same

Reply:
I just had to set `wsig = off` implicitly on instance:python27 because otherwise `bin/instance fg` shuts down quietly ...